ucm2/USB-Audio/MOTU/M4-HiFi-Stereo-Mono.conf

```diff
@@ -0,0 +1,139 @@
+Include.pcm_split.File "/common/pcm/split.conf"
+
+Macro [
+	{
+		SplitPCM {
+			Name "m4_stereo_out"
+			Direction Playback
+			Channels 2
+			HWChannels 4
+			HWChannelPos0 FL
+			HWChannelPos1 FR
+			HWChannelPos2 FC
+			HWChannelPos3 LFE
+		}
+	}
+	{
+		SplitPCM {
+			Name "m4_mono_in"
+			Direction Capture
+			Channels 1
+			HWChannels 4
+			HWChannelPos0 FL
+			HWChannelPos1 FR
+			HWChannelPos2 FC
+			HWChannelPos3 LFE
+		}
+	}
+]
+
+SectionDevice."Line1-2" {
+	Comment "Headphone + Monitor Out"
+	Value {
+		PlaybackPriority 200
+		PlaybackMixer "default:${CardId}"
```

Comment:
Remove the Playback/CaptureMixer lines unless you do something special with the mixer (remapping). See `ucm2/USB-Audio/Behringer/UMC204HD-HiFi.conf` and `/common/ctl/remap.conf`.

Reply:
Mixer entries are removed.